pysr/sr.py

```diff
@@ -997,6 +997,20 @@ def from_file(
             The model with fitted equations.
         """
 
+        class CustomUnpickler(pkl.Unpickler):
```

Comment:
Can this class have a more descriptive name?

Reply:
rename as `CrossPlatformPathUnpickler`